src/components/LeftSidebar/InvitationHandler.vue

```diff
@@ -228,14 +228,14 @@ export default {
 			padding-left: 4px;
 
 			&__name {
-				line-height: 1.2;
+				line-height: 1.3;
```

Comment:
`line-height` is inherited and not overwritten in `NcListItem`. We can put it directly on the `NcListItem` without `deep` overrides and without specifying each element manually with internal classes.

Reply:
Not in that case (not a NcListItem), but I got the idea